### ISSUE TITLE
Swift Package Manager support

### ISSUE THREAD

**Issue by patchthecode** [stripe/stripe-ios#1255]

**Comment by davidme-stripe** [comment]
Hey @patchthecode, thanks for filing this. We'd like to add support for Swift Package Manager, but we'll need them to support image resources[\[1\]](https://bugs.swift.org/browse/SR-2866) and embedded binaries[\[2\]](https://forums.swift.org/t/spm-support-for-binaries-distribution/25549/32)[\[3\]](https://forums.swift.org/t/distributing-binary-closed-source-libraries/23737/20) before we can get started.

**Comment by patchthecode** [comment]
understood.

**Comment by Andrewangeta** [comment]
@davidme-stripe @patchthecode  I have good news about resource support. Wanted to make sure I came back to this issue. 
https://forums.swift.org/t/draft-proposal-package-resources/29941

**Comment by itrack** [comment]
Any news?

**Comment by davidme-stripe** [comment]
Looks like a proposal for binary dependencies was recently accepted, but hasn't yet landed in Swift Package Manager: https://github.com/apple/swift-evolution/blob/master/proposals/0272-swiftpm-binary-dependencies.md

**Comment by Andrewangeta** [comment]
@Davidme-stripe looks like we have it finally implemented for swift 5.3 

https://github.com/apple/swift-evolution/commit/1b5b5b284aa7c9317fbe042414c3b3d56e7b4842?short_path=fe47ec9#diff-fe47ec96b7546e627fba976d86c143cc

**Comment by janekwunderlich** [comment]
Xcode 12 Beta introduces support for SPM 5.3, which now officially supports binary dependencies. Is there any chance we can try this out already?

https://github.com/apple/swift-evolution/blob/master/proposals/0272-swiftpm-binary-dependencies.md

**Comment by bdrelling** [comment]
@davidme-stripe Hey David, any news on this? We need a path forward with Xcode 12 just around the corner. Thanks.

**Comment by davidme-stripe** [comment]
I've been out on leave, sorry! I'll set aside some time to look at this next week.

**Comment by tush16ar555** [comment]
Okay on d wayy

Tushar Das

**Comment by bdrelling** [comment]
No need to apologize for being out on leave! Thank you for checking into it, I really appreciate it. 🙂

**Comment by davidme-stripe** [comment]
This has been merged into `master`. We'll do a formal release soon.

**Comment by davidme-stripe** [comment]
This is in SDK 20.0.0.
